sdk/tests/platform/test_az_mqtt5_policy.c

```diff
@@ -211,6 +216,7 @@ static void test_az_mqtt5_policy_init_success(void** state)
   ref_puback = 0;
   ref_recv = 0;
   ref_disconnect = 0;
+  expect_msg_properties = 0;
```

Comment:
does this need to be set here?

Reply:
I set all of the "reference" variables at the start of the test suite, but it is not necessary